pkg/controller/plan/restore.go

```diff
@@ -232,7 +232,16 @@ func (r *RestoreManager) DoPostReady(comp *component.SynthesizedComponent,
 	backupObj *dpv1alpha1.Backup) error {
 	jobActionLabels := constant.GetCompLabels(r.Cluster.Name, comp.Name)
 	if len(comp.Roles) > 0 {
-		jobActionLabels[instanceset.AccessModeLabelKey] = string(appsv1alpha1.ReadWrite)
+		// HACK: assume the role with highest priority to be writable
```

Comment:
It's better to let the data-protection define a policy to select target pods explicitly.

Reply:
Maybe we can add a new field in actionset to indicate how to select pod in restore's postready step. But @wangyelei suggests for now we can just hack it.